syz-fuzzer/fuzzer.go

```diff
@@ -83,19 +83,21 @@ const (
 	StatHint
 	StatSeed
 	StatCollide
+	StatBufferTooSmall
 	StatCount
 )
 
 var statNames = [StatCount]string{
-	StatGenerate:  "exec gen",
-	StatFuzz:      "exec fuzz",
-	StatCandidate: "exec candidate",
-	StatTriage:    "exec triage",
-	StatMinimize:  "exec minimize",
-	StatSmash:     "exec smash",
-	StatHint:      "exec hints",
-	StatSeed:      "exec seeds",
-	StatCollide:   "exec collide",
+	StatGenerate:       "exec gen",
+	StatFuzz:           "exec fuzz",
+	StatCandidate:      "exec candidate",
+	StatTriage:         "exec triage",
+	StatMinimize:       "exec minimize",
+	StatSmash:          "exec smash",
+	StatHint:           "exec hints",
+	StatSeed:           "exec seeds",
+	StatCollide:        "exec collide",
+	StatBufferTooSmall: "exec buffer_too_small",
```

Comment:
This string will be shown right in the syz-manager web ui. These strings don't use cpp_identifier_name_style and this is also not "exec", these are not executed and won't be counted in "exec total".
Please give it some other name.